Libraries/pbxbuild/Sources/Phase/ProductTypeResolver.cpp

```diff
@@ -335,6 +465,15 @@ resolve(Phase::Environment const &phaseEnvironment, Phase::Context *phaseContext
     }
 
     /*
+     * Create the product's module maps, if required.
+     */
+    if (pbxsetting::Type::ParseBoolean(environment.resolve("DEFINES_MODULE"))) {
```

Comment:
@compnerd could probably answer better than me. It's it's a file that does some stuff I don't really understand but (in this case) is used to to make groups of Objective-C headers into Swift modules.